### ISSUE TITLE
bug: Insufficient color contrast ratio

### ISSUE THREAD

**Issue by mejarc** [mejarc/mws-restaurant-stage-1#5]
* a
* h1#restaurant-name
* h2
* #footer

**Comment by mejarc** [comment]
a -- #005096

#restaurant-name -- #aa5a00

h2 -- #7f4200

#footer -- #b1b1b1

**Comment by mejarc** [comment]
#filter-options h2 --  background: #005e8f

#restaurants-list li h1  -- #b56000

#restaurants-list li a -- background to #7d4f00
